_posts/2018-11-01-managing-translations-with-crowdin.md

```diff
@@ -0,0 +1,148 @@
+
+Approver view:
+<img class="image fit" style="margin:0px auto; max-width: 1439px;" alt="Crowdin approver view" src="/img/2018-11-02-managing-translations-with-crowdin/approver_mode.png">
+
+## Key features
+* A large number of file formats are supported, ranging from `json` files to `csv` and `properties` files.
+It can thus be used for Angular, React, Vue ... applications as well as a Spring Boot service and so on.
+* Crowdin has a very clever parser and detects thing such as HTML tags and variables.
+It will warn you if they are translated and they no longer match the original tags/names.
+* Quality assurance checks will help the approver in verifying the translations.
+* Extra context can be provided for each source string to help translators understand what exactly they need to translate.
+* Integration with source control systems is very easy and will make sure that the source strings are up to date with your Git repository. 
+Translations will automatically be pushed to your Git repository.
+* There's even the possibility to integrate it with your website and have people do the translations on your website directly.
+That way the translators get a lot of context of what to translate.
+* If you're unable to do the translation for a certain language, 
+Crowdin allows you to easily hire professional translators!
+
+## Getting started
+### Account
+To start using Crowdin, you of course need an account at [Crowdin.com](https://crowdin.com). 
+There's a limited free trial. 
+After that you can choose from a variety of payment plans depending on the number of source strings and projects you'll have.
+
+Once logged in, you can create a new project. 
+You can choose the source language and the languages you wish to translate to. 
+Then go to the project's settings and then to `API`.
+There you'll find a project identifier and a API key. 
+We'll need those later when setting up Crowdin in our project.
+
+<img class="image fit" style="margin:0px auto; max-width: 744px;" alt="Crowdin project settings" src="/img/2018-11-02-managing-translations-with-crowdin/languages.png">
+
+### Setup in your project
+First we need to install the Crowdin CLI. 
+For Mac you can run `brew install crowdin`. 
+For Windows an [installer](https://support.crowdin.com/cli-tool/) is available.
+
+Once installed, open a terminal and go to the root of your project and run `crowdin generate`.
+This will generate a `crowdin.yml` file that is used to configure Crowdin in your project.
+In that file four things need to be changed:
+* `project_identifier`: the one you find under API on the Crowdin site
+* `api_key`: also under API
+* `source`: this should point to the source translation file (e.g. in an Angular app `/src/assets/i18n/en.json`)
+* `translation`: the translated files will end up here (e.g. `/src/assets/i18n/%two_letters_code%.json`)
```

Comment:
So if we adapt the English translations this will cause the source file (`en.json`) to be overwritten?